### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sudiptpa**
Add contribution guidelines, and error reporting way on the README.md

### CODE REVIEW

**Comment by sudiptpa** [comment]
@rotassator Accept this PR during the Hacktoberfest challenge within this October if possible.

File: README.md [hunk 1]
```diff
@@ -94,3 +94,16 @@ $response = $gateway->createSingleUseBankToken([
 
 $singleUseTokenId = $response->getData('singleUseTokenId');
 ```
+
+## Contributing
+
+Contributions are **welcome** and will be fully **credited**.
```
Comment (rotassator) [inline README.md:100]: I'm not clear on what is meant by "fully credited" here. Github will keep track of contributors, but there's no current plan to create a custom list of contributors in the repo as such.

Comment (sudiptpa) [reply README.md:100]: Its like being thankful for supporting the package, if you would like to give feedback, i can update the PR. If you are not that happy with the PR, i am happy with closing it.

Comment (sudiptpa) [reply README.md:100]: @rotassator what about licensing ? Is this package following MIT license or other ?

Comment (rotassator) [reply README.md:100]: There's a LICENCE file for that. :)
https://github.com/rotassator/omnipay-payway-restapi/blob/master/LICENSE

**Review by rotassator — APPROVED** [review]

**Pull request merged**